docs/tooling/pepsi.md

```diff
@@ -84,3 +84,18 @@ cargo install --path tools/pepsi
 ```
 
 and adding `~/.cargo/bin` to the `PATH`.
+
+## Remote Compile
+
+To use the remote compilation you need to create yourself an account on the rechenknecht. Therefore open an ssh connection to ```root@134.28.57.226```. There create a new account by ```adduser {name}``` and set a password with ```passwd {name}```, you will then be asked to type your password. 
+
+In the next step change to your user ssh connection ```{name}@134.28.57.226```. There clone the HULKs repository using https: ```https://github.com/HULKs/hulk.git```.
```

Comment:
"Terminate the root ssh session and ..."

Reply:
Done